main_test.go

```diff
@@ -93,6 +93,38 @@ testdata/singlelevel/main.go: is missing the license header
 testdata/singlelevel/wrapper.go: is missing the license header
 testdata/singlelevel/zrapper.go: is missing the license header
 testdata/x-pack/wrong.go: is missing the license header
+testdata/x-pack-v2/correct.go: is missing the license header
+testdata/x-pack-v2/wrong.go: is missing the license header
+`[1:],
+		},
+		{
+			name: "Run a diff prints a list of files that need the Elastic license 2.0 header",
+			args: args{
+				args:     []string{"testdata"},
+				license:  "Elasticv2",
+				licensor: defaultLicensor,
+				ext:      defaultExt,
+				dry:      true,
+			},
+			want: 1,
+			err:  &Error{code: 1},
+			wantOutput: `
+testdata/cloud/doc.go: is missing the license header
+testdata/cloud/wrong.go: is missing the license header
+testdata/excludedpath/file.go: is missing the license header
+testdata/multilevel/doc.go: is missing the license header
+testdata/multilevel/main.go: is missing the license header
+testdata/multilevel/sublevel/autogen.go: is missing the license header
+testdata/multilevel/sublevel/doc.go: is missing the license header
+testdata/multilevel/sublevel/partial.go: is missing the license header
+testdata/singlelevel/doc.go: is missing the license header
+testdata/singlelevel/main.go: is missing the license header
+testdata/singlelevel/wrapper.go: is missing the license header
+testdata/singlelevel/zrapper.go: is missing the license header
+testdata/x-pack/doc.go: is missing the license header
+testdata/x-pack/wrong.go: is missing the license header
+testdata/x-pack-v2/doc.go: is missing the license header
```

Comment:
Is it a mistake that testdata/x-pack-v2/doc.testdata does not have the ELv2 license header? Why do we have "correct.testdata", "doc.testdata", and "wrong.testdata" in x-pack-v2, but only "doc.testdata" and "wrong.testdata" in x-pack?

Reply:
actually, I meant to remove doc and leave only `correct` and `wrong`. I'll update the PR.